docs/source/_templates/home.html

```diff
@@ -437,63 +488,78 @@ <h3 class="h-card__title">Mathematics</h3>
+                                            src="https://github.com/fury-gl/fury-communication-assets/raw/main/helical_path.gif"
+                                            alt="Helical path" align="middle"></a>
+                                            
                                 </div>
-                            </div>
-                            <div>
-                                <p>FURY allows you to create easily  multiple physics animations.</p>
                             </div>
                         </div>
                     </li>
                     <li class="molecular-dynamics">
                         <div class="col-sm">
-                            <p class="txt-center">Some Molecular Dynamics applications using FURY.</p>
+                            <p class="tabs-info txt-center">Some Molecular Dynamics applications using FURY.</p>
                         </div>
                         <div class="grid-container">
                             <div class="visualization-images">
-                                <div>
-                                    <div>
+                                <div class="image-grid grid g-gap-2 g-col-2">
+                                    <div class="g-span-2 molecular-band">
                                         <a href="fury-pybullet.html"><img src="https://furiousatoms.com/_static/images/fullerene_1.png" alt="fullrene Modellin" align="middle"></a>
                                     </div>
-                                    <div class="row">
-                                        <div class="col-sm p-t-20">
-                                            <a href="fury-pybullet.html"><img src="https://github.com/fury-gl/fury-communication-assets/raw/main/molecular_ribbons.gif" alt="Molecular Ribbon" align="middle"></a>
-                                        </div>
-                                        <div class="col-sm p-t-20">
-                                            <a href="fury-pybullet.html"><img src="https://github.com/fury-gl/fury-communication-assets/raw/main/molecular_surface.gif" alt="Molecular Surface" align="middle"></a>
-                                        </div>
+                                    <div>
+                                        <a href="fury-pybullet.html"><img
+                                                src="https://github.com/fury-gl/fury-communication-assets/raw/main/molecular_ribbons.gif"
+                                                alt="Molecular Ribbon" align="middle"></a>
+                                    </div>
+                                    <div>
+                                        <a href="fury-pybullet.html"><img
+                                                src="https://github.com/fury-gl/fury-communication-assets/raw/main/molecular_surface.gif"
+                                                alt="Molecular Surface" align="middle"></a>
```

Comment:
same as above

Reply:
This cannot be converted to raw user link or permalink as the size of the asset is too big